tenets/codelingo/php/unused-private-functions/codelingo.yaml

```
@@ -0,0 +1,16 @@
+tenets:
+  - flows:
+      codelingo/review:
+        comment: |
+          There is no call to this private function 
+    name: unused-private-functions
+    query: |
+      import codelingo/ast/php
+      php.file(depth = any):
+        php.stmt_class(depth=any):
```

```suggestion
        php.stmt_class(depth = any):
```
personal preference 😛